explorer/nodewatch/nodewatch/__init__.py

```diff
@@ -46,6 +46,10 @@ def create_app():
 	def index():  # pylint: disable=unused-variable
 		return redirect(url_for('symbol_summary'))
 
+	@app.route('/api/openapi')
```

Comment:
Can you also write a unit test for this?

Reply:
Added test for this endpoint.